<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QDialog" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="LeftSide_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
    background-color: #2c3e50;  /* Dark blue-gray */
    color: white;               /* Text color */
    font-weight: bold;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DASHBOARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reserveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve Room</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Settings_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>310</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Light Fixture broken</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roomButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rooms</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="RightSide_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>181</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
    
    background-color: #d0e9ff;   /* Light blue */
    color: #0d47a1;              /* Dark blue text */
    font-weight: bold;
    font-size: 18px;
    padding: 10px;
    border-radius: 12px;
    border: none;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>AC not working 406</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>400</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Door Lock malfunction</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="RA_label_9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Buildings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maintenanceButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QLabel" name="MaintenanceRequest_label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #ffe6cc;   /* Peach */
    color: #e65100;              /* Burnt orange text */
    font-weight: bold;
    font-size: 18px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>250</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RA_label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Room Allocation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Sink leak in room 101</string>
   </property>
  </widget>
  <widget class="QLabel" name="RA_label_11">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>160</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Requests</string>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>420</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>380</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QToolButton" name="OverviewButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QFrame" name="Chart_frame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>571</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Chart_frame {
    background-color: #ffffff;              /* White background */
    border: 2px solid #d3d3d3;              /* Light gray border */
                      /* Rounded corners */
    box-shadow: 0px 0px 10px rgba(0,0,0,0.2); /* Subtle shadow (if supported) */
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chart_plainTextEdit_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: #3498db;   /* Soft dashboard blue */
    border: none;
    border-radius: 4px;
    color: transparent;          /* Hide text */
}
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chart_plainTextEdit_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: #3498db;   /* Soft dashboard blue */
    border: none;
    border-radius: 4px;
    color: transparent;          /* Hide text */
}
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chart_plainTextEdit_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>61</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: #3498db;   /* Soft dashboard blue */
    border: none;
    border-radius: 4px;
    color: transparent;          /* Hide text */
}
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chart_plainTextEdit_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: #3498db;   /* Soft dashboard blue */
    border: none;
    border-radius: 4px;
    color: transparent;          /* Hide text */
}
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chart_plainTextEdit_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: #3498db;   /* Soft dashboard blue */
    border: none;
    border-radius: 4px;
    color: transparent;          /* Hide text */
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="RA_label_10">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>140</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QLabel" name="OccupiedRoom_label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #d6f5e3;   /* Aqua green */
    color: #2e7d32;              /* Forest green text */
    font-weight: bold;
    font-size: 16px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Occupied Room</string>
   </property>
  </widget>
  <widget class="QLabel" name="RA_label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Room Allocation</string>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="TotalRoom_label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #c2f0e0;   /* Soft mint green */
    color: #00695c;              /* Deep green text */
    font-weight: bold;
    font-size: 18px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Total Rooms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalStudent_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #d0e9ff;   /* Light blue */
    color: #0d47a1;              /* Dark blue text */
    font-weight: bold;
    font-size: 18px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Total Students</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>350</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>No hot water room 209</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    
    color: #0d47a1;              /* Dark blue text */
    font-weight: bold;
    font-size: 20px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>120</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
   
    color: #00695c;              /* Deep green text */
    font-weight: bold;
    font-size: 20px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>67</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
   
    color: #00695c;              /* Deep green text */
    font-weight: bold;
    font-size: 20px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    
    color: #e65100;              /* Burnt orange text */
    font-weight: bold;
    font-size: 20px;
    padding: 10px;
    border-radius: 12px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <zorder>plainTextEdit</zorder>
  <zorder>RightSide_plainTextEdit</zorder>
  <zorder>textEdit</zorder>
  <zorder>LeftSide_plainTextEdit</zorder>
  <zorder>DashName_label_8</zorder>
  <zorder>DashName_label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>reserveButton</zorder>
  <zorder>Settings_Button</zorder>
  <zorder>label_13</zorder>
  <zorder>DashName_label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>roomButton</zorder>
  <zorder>label_16</zorder>
  <zorder>label_15</zorder>
  <zorder>label_3</zorder>
  <zorder>label_7</zorder>
  <zorder>RA_label_9</zorder>
  <zorder>maintenanceButton</zorder>
  <zorder>MaintenanceRequest_label_4</zorder>
  <zorder>RA_label_8</zorder>
  <zorder>label_12</zorder>
  <zorder>RA_label_11</zorder>
  <zorder>line_16</zorder>
  <zorder>label_4</zorder>
  <zorder>label_9</zorder>
  <zorder>label_5</zorder>
  <zorder>line_15</zorder>
  <zorder>label</zorder>
  <zorder>OverviewButton</zorder>
  <zorder>label_11</zorder>
  <zorder>Chart_frame</zorder>
  <zorder>RA_label_10</zorder>
  <zorder>OccupiedRoom_label_3</zorder>
  <zorder>RA_label_7</zorder>
  <zorder>line_14</zorder>
  <zorder>TotalRoom_label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_10</zorder>
  <zorder>TotalStudent_label</zorder>
  <zorder>DashName_label_7</zorder>
  <zorder>line_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
